Rebase Nadja's json updates to dev branch

Kjili opened the pull request:
Habe ein paar Korrekturen wg Rechtschreibungen/Grammatik eingefügt. Rechtschreibung und Wasserspeicher statt reservoir.

Spexify commented on godot/JSONs/scenarios.json:
```diff
@@ -33,18 +33,15 @@
-		"title": "Waldbrand\nStarke Rauchentwicklung",
-		"description": "[img]res://assets/GUI/icons/forestfire_icon.png[/img][font_size=30][b]\t\t\tWaldbrand[center]Starke Rauchentwicklung[/center][/b][/font_size]\n\nWarnung vor großflächigem Brand im Waldgebiet Gluthain. In der Folge kann es zu Störungen oder dem Ausfall verschiedener Infrastruktursysteme kommen.\n\n[img]res://assets/GUI/icons/icon_pin_info.png[/img][b]Handlungsempfehlung:[/b]\nMei ...
+		"icon_id": 0,
+		"description": ""
```
Why is the description here deleted?

Spexify commented on godot/JSONs/scenarios.json:
```diff
@@ -33,18 +33,15 @@
+		"description": ""
+	},
+	"Hochwasser": {
+		"icon_id": 1,
```
Why is the icon_id here changed?

Spexify commented on godot/JSONs/scenarios.json:
```diff
@@ -33,18 +33,15 @@
+		"description": "[img]res://assets/GUI/icons/_hochwasser-export.png[/img][font_size=30][b]Hochwasser[/b][/font_size]\n\nMomentan herrscht Hochwasser. In den folgenden Tagen können verschiedene Infrastrukturen ausfallen. Dies beinhaltet [b]Stromausfall[/b], [b]Überschwemmungen[/b], [b]Wasserverschmutzung[/b] oder [b]Ausfall[/b] sowie [b]Ausfall des Mobilefunknetzes[/b]\n\n[img]res://assets/GUI/ic ...
+	},
+	"Tutorial": {
+		"icon_id": 3,
```
Should be icon_id 7

NajaNadja committed:
Update jsons

Habe ein paar Korrekturen wg Rechtschreibungen/Grammatik eingefügt.
Rechtschreibung und Wasserspeicher statt reservoir.

NajaNadja committed:
Update scenarios.json

Alle BBK-Links geprüft und aktualisiert

Spexify committed:
Merge branch 'dev' into json_update_nadja
